Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/BlobTriggerCSharp.cs

```diff
@@ -1,23 +1,25 @@
-using System;
 using System.IO;
+using System.Threading.Tasks;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 
 namespace Company.Function
 {
     public class BlobTriggerCSharp
     {
-        private readonly ILogger _logger;
+        private readonly ILogger<BlobTriggerCSharp> _logger;
 
-        public BlobTriggerCSharp(ILoggerFactory loggerFactory)
+        public BlobTriggerCSharp(ILogger<BlobTriggerCSharp> logger)
         {
-            _logger = loggerFactory.CreateLogger<BlobTriggerCSharp>();
+            _logger = logger;
         }
 
-        [Function("BlobTriggerCSharp")]
-        public void Run([BlobTrigger("PathValue/{name}", Connection = "ConnectionValue")] string myBlob, string name)
+        [Function(nameof(BlobTriggerCSharp))]
```

Comment:
Now that we are no longer using Client; we should keep the name in the path and use that in the example, same for fsharp